core/modules/node/node.module

```diff
@@ -1664,7 +1664,7 @@ function node_menu() {
   foreach (node_type_get_types() as $type) {
     $type_url_str = str_replace('_', '-', $type->type);
     $items['node/add/' . $type_url_str] = array(
-      'title' => $type->name,
+      'title' => 'Add content: ' . $type->name,
```

Comment:
...same as the menu items case. This also changes the actual menu items from:

*Content* -> *Add content* -> *[content_type_name]*

to:

*Content* -> *Add content* -> *Add content: [content_type_name]*

Not good. We need to change **only** the search results by inserting the "Add content:" part between the "Content: " and "[content_type_name]" parts.